src/test/e2e/51_oci_compose_test.go

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package test provides e2e tests for Zarf.
+package test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/pkg/utils/exec"
+	dconfig "github.com/docker/cli/cli/config"
+	"github.com/stretchr/testify/suite"
+	"oras.land/oras-go/v2/registry"
+)
+
+type SkeletonSuite struct {
+	suite.Suite
+	Remote    *utils.OrasRemote
+	Reference registry.Reference
+}
+
+var (
+	importEverything   = filepath.Join("src", "test", "test-packages", "51-import-everything")
+	importception      = filepath.Join("src", "test", "test-packages", "51-import-everything", "inception")
+	everythingExternal = filepath.Join("src", "test", "test-packages", "everything-external")
+	absNoCode          = filepath.Join("/", "tmp", "nocode")
+)
+
+func (suite *SkeletonSuite) SetupSuite() {
+	err := os.MkdirAll(filepath.Join("src", "test", "test-packages", "51-import-everything", "charts"), 0755)
+	suite.NoError(err)
+	err = utils.CreatePathAndCopy(filepath.Join("examples", "helm-local-chart", "chart"), filepath.Join("src", "test", "test-packages", "51-import-everything", "charts", "local"))
+	suite.NoError(err)
+	suite.DirExists(filepath.Join("src", "test", "test-packages", "51-import-everything", "charts", "local"))
+
+	err = utils.CreatePathAndCopy(importEverything, everythingExternal)
+	suite.NoError(err)
+	suite.DirExists(everythingExternal)
```

Comment:
Can we add this to .gitignore so that it doesn't get copied if there is a failed/borked test locally.